packages/fe-fpm-writer/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@sap-ux/fe-fpm-writer",
+    "version": "0.1.0",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "pnpm clean && tsc",
+        "clean": "rimraf dist",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "watch": "tsc --watch"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "templates"
+    ],
+    "dependencies": {
+        "ejs": "3.1.6",
+        "mem-fs": "2.1.0",
+        "mem-fs-editor": "9.3.0",
```

Comment:
Consolidating the version should be done in a separate PR as part of #170. We should get all modules to `9.3.0` since we are using the `fs.dump` function in many tests which is working in `9.0.0` but not documented (and part of the types)

Reply:
As discussed this is fine. We can update the other to 9.3.0 in a week or two.